README.md

```diff
@@ -24,7 +24,5 @@ Smile is a comprehensive learning machine developed for Java and Scala, however
 
 Smile is a comprehensive learning machine. It is built to essentially teach AI using statistical precedures, functions and the like. This is way above my understanding on exactly how they work but this kind of thing interests me as I have wanted to try my hand at making a bot in Rocket League using some sort of machine learning. So on an extremely low level of explaination it is used when you are wanting to implement methods to train and teach your AI. This is then tracked using graphs, maps, and other scaling to show your progress on the AI. However, this is really where I start to not comprehend what is going on.
 
-**Note:** All your work should be *in your own words*. If you quote anyone, or if you use any resources other than the Kotlin Overview docs you should say so and provide a link in a **Bibliography**.
```

Comment:
Removed my instructions. Make sure to remove anything like this when you write your README.md